### PR TITLE
Bugfix url option is missing for edit-config

### PR DESCRIPTION
**Pull request by fredgan**
Hi, 
This PR adds a `url` option to `edit-config`, as mentioned at #431. :-)

### CODE REVIEW

File: ncclient/operations/edit.py [hunk 1]
```diff
@@ -66,6 +66,12 @@ def request(self, config, format='xml', target='candidate', default_operation=No
         if format == 'text':
```
Comment (einarnn) [inline ncclient/operations/edit.py:66]: can you change `if` to `elif`?

File: ncclient/operations/edit.py [hunk 2]
```diff
@@ -66,6 +66,12 @@ def request(self, config, format='xml', target='candidate', default_operation=No
...
             config_text = sub_ele(node, "config-text")
             sub_ele(config_text, "configuration-text").text = config
+        if format == 'url':
```
Comment (einarnn) [inline ncclient/operations/edit.py:69]: can you change `if` to `elif`?

File: ncclient/operations/edit.py [hunk 3]
```diff
@@ -66,6 +66,12 @@ def request(self, config, format='xml', target='candidate', default_operation=No
...
+            if "://" in config:
```
Comment (einarnn) [inline ncclient/operations/edit.py:70]: You should probably use https://docs.python.org/3/library/urllib.parse.html to check for URL validity.

**Review by einarnn — CHANGES_REQUESTED** [review]
Please see review comments.

**Comment by fredgan** [comment]
@einarnn Hi Einar, They have been modified. Please review it again.

**Review by einarnn — APPROVED** [review]

**Pull request merged**